### DISCUSSION TITLE
FileUpload with custom livewire component is not rendering properly.

### DISCUSSION THREAD

**Discussion by rajentrivedi** [filamentphp/filament#12051] [Help]
### Package

Form builder

### Package Version

v3.2

### How can we help you?

I am trying to insert FileUpload class with custom livewire component, it is giving me 
![Screenshot from 2024-03-27 11-07-37](https://github.com/filamentphp/filament/assets/69707769/995fbdc3-313a-497c-ac07-393401e25a32)

`public function form(Form $form): Form
    {
        return $form
            ->schema([
                TextInput::make('title'),
                ColorPicker::make('test'),
                FileUpload::make('log')
            ])
            ->statePath('data')
            ->model(Job::class);
    }`
    
    where I am doing mistake, please help me in this regard.
    
    Thank You.

**Comment by leandrocfe** [comment]
Did you follow this section?

https://filamentphp.com/docs/3.x/forms/adding-a-form-to-a-livewire-component/

Did you add the mount method?

Please share the whole code.
